index.html

```diff
@@ -81,148 +81,155 @@ <h4 class="list-heading">Skills</h4>
 
             </div>
         </section>
-
+       <section>
         <div class="link-page">
             <hr>
-
             <ul class="icon">
+                <li>
+                    <p class="work-left">Work</p>
+                </li>
                 <li><img src="images/github1.png" alt="github" class="icons"></li>
                 <li><img src="images/linkden.png" alt="linkden1" class="icons "><img src="images/linkden2.png"
                         alt="linkden1" class="icons "></li>
                 <li><img src="images/1.png" alt="1" class="icons"></li>
                 <li><img src="images/twitter1.png" alt="twitter" class="icons"></li>
                 <li><img src="images/2.png" alt="2" class="icons"></li>
+                <li>
+                    <p class="work-right">Work</p>
+                </li>
```

Comment:
- [ ] Please  🙏🏽 review the source of the icons to show them in the browser:

> You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

![image](https://user-images.githubusercontent.com/58642949/187029213-751a3c20-c909-4eed-ab77-767bdf9b9002.png)